docs/reference-guides/monitoring-v2-configuration/examples.md

```diff
@@ -8,20 +8,22 @@ title: Examples
 
 ### ServiceMonitor
 
-An example ServiceMonitor custom resource can be found [here.](https://github.com/prometheus-operator/prometheus-operator/blob/master/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml)
+See the official prometheus-operator Github for an example [ServiceMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/master/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml) YAML.
 
 ### PodMonitor
 
-An example PodMonitor can be found [here.](https://github.com/prometheus-operator/prometheus-operator/blob/master/example/user-guides/getting-started/example-app-pod-monitor.yaml) An example Prometheus resource that refers to it can be found [here.](https://github.com/prometheus-operator/prometheus-operator/blob/master/example/user-guides/getting-started/prometheus-pod-monitor.yaml)
+See the official prometheus-operator Github for an example [PodMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/master/example/user-guides/getting-started/example-app-pod-monitor.yaml) YAML.
+
+Refer to the official prometheus-operator Github for an [example Prometheus resource](https://github.com/prometheus-operator/prometheus-operator/blob/master/example/user-guides/getting-started/prometheus-pod-monitor.yaml) that refers to a PodMonitor.
```

Comment:
```suggestion
See the [Prometheus Operator documentation](https://prometheus-operator.dev/docs/user-guides/getting-started/#using-podmonitors) for an example PodMonitor and an example Prometheus resource that refers to a PodMonitor.
```

- It seems unnecessary to have the "see/refer to the official PO..." phrasing twice in the same section when they both point to the same reference site.
- Not sure if they existed at the time, but there are now corresponding docs for the links used.